packages/react-components/src/templates/SharedResearchOutput.tsx

```diff
@@ -52,72 +80,153 @@ const SharedResearchOutput: React.FC<SharedResearchOutputProps> = ({
   asapFunded,
   usedInPublication,
   pmsEmails,
+  doi,
+  rrid,
+  accession,
+  labCatalogNumber,
   ...props
-}) => (
-  <div css={containerStyles}>
-    <BackLink href={backHref} />
-    <div css={cardsStyles}>
-      <SharedResearchOutputHeaderCard {...props} />
-      {(description || !!tags.length) && (
-        <Card>
-          {description && (
+}) => {
+  const isLabCatalogNumberALink =
```

Comment:
I don't like this. I would do something like 
```suggestion
  const isLabCatalogNumberALink = ()=> {
  try {
    url = new URL(string);
  } catch (_) {
    return false;  
  }
  return true; 
}
```

and stick it in packages/react-components/src/utils.ts